modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/schedule_modal.jsp

```diff
@@ -13,7 +13,9 @@ KBArticle kbArticle = KBArticleServiceUtil.getLatestKBArticle(ParamUtil.getLong(
 String displayDateString = StringPool.BLANK;
 
 if (kbArticle.getDisplayDate() != null) {
-	displayDateString = dateFormatDateTime.format(kbArticle.getDisplayDate());
+	Format displayFormatDate = FastDateFormatFactoryUtil.getSimpleDateFormat("yyyy-MM-dd HH:mm", locale, timeZone);
```

Comment:
Should this date format locale dependent?

Reply:
What is your suggestion here, remove the locale dependency?